motioncorr/requirements.txt

```diff
@@ -0,0 +1 @@
+scipion-em
```

Comment:
why this file is inside motioncorr folder? shouldn't it be outside?

Reply:
Also, I would put scipion-em requirement directly into setup.py instead of creating more files

Reply:
You are right, my bad. It was meant to go to the root folder.

I can undo that part if you want.